ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/restore/KsqlRestoreCommandTopic.java

```diff
@@ -427,13 +440,20 @@ private static void maybeCleanUpQuery(final byte[] command, final KsqlConfig ksq
         System.out.println(
             String.format(
                 "Cleaned up internal state store and internal topics for query %s",
-                applicationId));
+                topicPrefix));
       } catch (final Exception e) {
-        System.out.println(String.format("Failed to clean up query %s ", applicationId));
+        System.out.println(String.format("Failed to clean up query %s ", topicPrefix));
```

Comment:
Wonder why we use `System.out.println` here instead of log4j? Any of @rodesai @agavra @vcrfxia know if this is intentional?

Reply:
@ableegoldman The restore cmd topic tool is run via the cli I imagine that it is used there